[Bug]: ldid: cannot execute binary file: Exec format error

### What are the steps to reproduce this issue?

1. Make new Project
2. Run `make package`

### What happens?
I followed this tutorial: https://theos.dev/docs/installation-linux and then I tried to compile for the first time a project, but an ldid error occured, although everythin is installed correctly... ldid seems to be the problem.

### What were you expecting to happen?
compiling

### Paste any relevant logs, error output, etc.
```shell
make package
> Making all for tweak djayPro…
==> Merging tweak djayPro…
==> Signing djayPro…
bash: line 1: /home/chris/theos/toolchain/linux/iphone/bin/ldid: cannot execute binary file: Exec format error
make[2]: *** [/home/chris/theos/makefiles/instance/library.mk:51: /home/chris/theosProjects/djaypro/.theos/obj/debug/djayPro.dylib] Error 126
rm /home/chris/theosProjects/djaypro/.theos/obj/debug/djayPro.dylib.4e292e05.unsigned
make[1]: *** [/home/chris/theos/makefiles/instance/library.mk:37: internal-library-all_] Error 2
make: *** [/home/chris/theos/makefiles/master/rules.mk:119: djayPro.all.tweak.variables] Error 2
```


### Additional context
_No response_

### What OS are you seeing the problem on?
Linux subsystem of Windows

### What OS version does it have installed?
Ubuntu 22.04.1 LTS (on Windows 10 64x)

### What toolchain and version are you using?
clang

### Which SDK version are you using?
iOS 14.5 (don't know, where to get a iOS 15 SDK ^^)

### What OS is your client device running?
iOS

### What OS version does it have installed?
iOS 15

### Which device model is it?
iPad Air 4th gen.

### Which jailbreak is it using (if relevant)?
XinaA15

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -134,7 +134,7 @@ get_sdks() {
 	if [[ -d $THEOS/sdks/ && $(ls -A "$THEOS/sdks/" | grep sdk) ]]; then
 		update "SDKs appear to already be installed."
 	else
-		update "SDKS do not appear to be installed. Installing now..."
+		update "SDKs do not appear to be installed. Installing now..."
 		curl -L https://api.github.com/repos/theos/sdks/tarball -o sdks.tar.gz
 		TMP=$(mktemp -d)
 		tar -xvf sdks.tar.gz --strip=1 -C $TMP

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -58,10 +58,6 @@ elif [[ $CSHELL == zsh ]]; then
 # elif [[ $CSHELL == csh ]]; then
 # 	SHELL_ENV="$HOME/.cshrc"
 fi
-NONINTERACTIVE=0
-if ! [[ -z $CI ]]; then
-	NONINTERACTIVE=1
-fi
 
 
 # The work
@@ -97,7 +93,7 @@ set_theos() {
 				update "'/opt' already exists. Checking its ownership..."
 				# Check that '/opt' isn't owned by root
 				OWNER="$(stat -c '%U' /opt)"
-				if [[ ${OWNER,,} == root ]]; then
+				if [[ $OWNER == root ]]; then
 					update "Owner of '/opt' is root. Attempting to switch owner to mobile..."
 					sudo chown mobile /opt \
 						&& update "Owner of '/opt' successfully transfered to mobile from root!" \
@@ -365,7 +361,7 @@ linux() {
 	else
 		update "A toolchain does not appear to be installed."
 		stoolchain="n"
-		if [[ $NONINTERACTIVE -eq 0 ]]; then
+		if [[ -z $CI ]]; then
 			read -p "Would you like your toolchain to support Swift (larger toolchain size) or not (smaller toolchain size)? [y/n]" stoolchain
 		fi
 		if theos_bool $stoolchain; then
@@ -480,12 +476,13 @@ linux() {
 
 # 	# Get a toolchain
 # 	update "Checking for iOS toolchain..."
+# 	# TODO: FreeBSD unique toolchain path != /linux/ ?
 # 	if [[ -d $THEOS/toolchain/linux/iphone/ && $(ls -A "$THEOS/toolchain/linux/iphone") ]]; then
 # 		update "A toolchain appears to already be installed."
 # 	else
 # 		update "A toolchain does not appear to be installed."
 # 		stoolchain="n"
-# 		if [[ $NONINTERACTIVE -eq 0 ]]; then
+#		if [[ -z $CI ]]; then
 # 			read -p "Would you like your toolchain to support Swift (larger toolchain size) or not (smaller toolchain size)? [y/n]" stoolchain
 # 		fi
 # 		if theos_bool $stoolchain; then
@@ -498,16 +495,12 @@ linux() {
 # 			ln -s $THEOS/toolchain/linux/iphone $THEOS/toolchain/swift
 # 			rm -r swift-5.3.2-RELEASE-ubuntu20.04.tar.zst $TMP
 # 		else
-# 			curl -LO https://github.com/sbingner/llvm-project/releases/latest/download/linux-ios-arm64e-clang-toolchain.tar.lzma
-# 			TMP=$(mktemp -d)
-# 			tar -xvf linux-ios-arm64e-clang-toolchain.tar.lzma -C $TMP
-# 			mkdir -p $THEOS/toolchain/linux/iphone
-# 			mv $TMP/ios-arm64e-clang-toolchain/* $THEOS/toolchain/linux/iphone/
-# 			rm -r linux-ios-arm64e-clang-toolchain.tar.lzma $TMP
+#			curl -LO https://github.com/L1ghtmann/llvm-project/releases/latest/download/iOSToolchain.tar.xz
+#			tar -xvf iOSToolchain.tar.xz -C $THEOS/toolchain/
+#			rm iOSToolchain.tar.xz
 # 		fi
 
 # 		# Confirm that toolchain is usable
-# 		# TODO: FreeBSD unique toolchain path != /linux/ ?
 # 		if [[ -x $THEOS/toolchain/linux/iphone/bin/clang ]]; then
 # 			update "Successfully installed the toolchain!"
 # 		else

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -340,7 +340,7 @@ linux() {
 
 	# Check for WSL
 	update "Checking for WSL..."
-	if [[ $(uname -r | sed -n 's/.*\( *Microsoft *\).*/\1/ip') == microsoft ]]; then
+	if [[ $(uname -r | sed -n 's/.*\( *Microsoft *\).*/\L\1/ip') == microsoft ]]; then
 		VERSION=$(uname -r | sed 's/.*\([[:digit:]]\)[[:space:]]*/\1/')
 		if [[ $VERSION -eq 1 ]]; then
 			update "WSL1! Need to fix fakeroot..."

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -153,8 +153,15 @@ get_sdks() {
 darwin() {
 	# Check for Xcode
 	XCODE="$(xcode-select -p)"
-	if [[ $XCODE != *.app/Contents/Developer ]]; then
+	if [[ $XCODE == /Library/Developer/CommandLineTools && ! -d /Applications/Xcode.app/Contents/Developer/ ]]; then
 		error "Xcode, not just the Command Line Tools, is required for Theos to function properly. Please install Xcode before continuing with the installation."
+		common "We recommend that you install Xcode from https://developer.apple.com/download/applications/ instead of from the Mac App Store as it's much faster."
+		exit 3
+	elif [[ $XCODE == /Library/Developer/CommandLineTools && -d /Applications/Xcode.app/Contents/Developer/ ]]; then
+		common "Xcode developer directory is currently $XCODE; switching to /Applications/Xcode.app/Contents/Developer/..."
+		sudo xcode-select -s /Applications/Xcode.app/Contents/Developer/
+	elif [[ $XCODE != *.app/Contents/Developer ]]; then
+		error "Xcode is required for Theos to function properly. Please install Xcode before continuing with the installation."
 		common "We recommend that you install Xcode from https://developer.apple.com/download/applications/ instead of from the Mac App Store as it's much faster."
 		common "If you already have Xcode installed, check the output of 'xcode-select -p'; you may need to change your developer directory via 'sudo xcode-select -s <path>'."
 		exit 3

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -431,12 +431,9 @@ linux() {
 					sudo zypper install -y libncurses5
 					;;
 			esac
-			curl -LO https://github.com/sbingner/llvm-project/releases/latest/download/linux-ios-arm64e-clang-toolchain.tar.lzma
-			TMP=$(mktemp -d)
-			tar -xvf linux-ios-arm64e-clang-toolchain.tar.lzma -C $TMP
-			mkdir -p $THEOS/toolchain/linux/iphone
-			mv $TMP/ios-arm64e-clang-toolchain/* $THEOS/toolchain/linux/iphone/
-			rm -r linux-ios-arm64e-clang-toolchain.tar.lzma $TMP
+			curl -LO https://github.com/L1ghtmann/llvm-project/releases/latest/download/iOSToolchain.tar.xz
+			tar -xvf iOSToolchain.tar.xz -C $THEOS/toolchain/
+			rm iOSToolchain.tar.xz
 		fi
 
 		# Confirm that toolchain is usable

--- a/bin/install-theos
+++ b/bin/install-theos
@@ -134,7 +134,7 @@ get_theos() {
 
 get_sdks() {
 	# Get patched sdks
-	update "Checking for patched sdks..."
+	update "Checking for patched SDKs..."
 	if [[ -d $THEOS/sdks/ && $(ls -A "$THEOS/sdks/" | grep sdk) ]]; then
 		update "SDKs appear to already be installed."
 	else
@@ -160,6 +160,7 @@ darwin() {
 	if [[ $XCODE != *.app/Contents/Developer ]]; then
 		error "Xcode, not just the Command Line Tools, is required for Theos to function properly. Please install Xcode before continuing with the installation."
 		common "We recommend that you install Xcode from https://developer.apple.com/download/applications/ instead of from the Mac App Store as it's much faster."
+		common "If you already have Xcode installed, check the output of 'xcode-select -p'; you may need to change your developer directory via 'sudo xcode-select -s <path>'."
 		exit 3
 	fi
 

